src/Nokia_LCD.cpp

```diff
@@ -14,9 +14,18 @@
 namespace {
 const uint8_t kDisplay_max_width = 84;
 const uint8_t kDisplay_max_height = 48;
+
+// Instantiate the default font
+const LcdFont nokiaFont { 
+    [](char c) { return Nokia_LCD_Fonts::kDefault_font[c - 0x20]; }, 
+    Nokia_LCD_Fonts::kColumns_per_character,
+    Nokia_LCD_Fonts::hSpace, 
+    1 
+};
+
 // Each row is made of 8-bit columns
 const unsigned int kTotal_rows =
-    kDisplay_max_height / Nokia_LCD_Fonts::kRows_per_character;
+    kDisplay_max_height / LcdFont::ROWS_PER_CHARACTER;
```

Comment:
As explained above, basically just undo this change and remove the added static field.

Reply:
Done

Reply:
Did you forget to push? :scream: